From: Martin Pitt <martin.pitt@ubuntu.com>
Date: Mon, 9 Feb 2015 10:53:43 +0100
Subject: Only start logind if dbus is installed

logind fails to start in environments without dbus, such as LXC containers or
servers. Add a startup condition to avoid the very noisy startup failure.

Part of #772700
---
 units/systemd-logind.service.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/units/systemd-logind.service.in b/units/systemd-logind.service.in
index f131b60..952a07c 100644
--- a/units/systemd-logind.service.in
+++ b/units/systemd-logind.service.in
@@ -16,6 +16,7 @@ Documentation=man:org.freedesktop.login1(5)
 
 Wants=user.slice modprobe@drm.service
 After=nss-user-lookup.target user.slice modprobe@drm.service
+ConditionPathExists=/lib/systemd/system/dbus.service
 
 # Ask for the dbus socket.
 Wants=dbus.socket
